admin/afs-client-accessd/afs-client-accessd

```diff
@@ -1230,6 +1245,27 @@ spawn_exporter()
 	exit(0);
 }
 
+sub
+time_to_export()
+{
```

Comment:
After thinking about this a bit, I think this function is not clear to me because it is doing several different things.  I feel it would be more clear if we had functions which had more clear intent. 

For example, for this feature we need to do the following, which can be though of as functions. Each take arguments which evaluate to one (and only one) result value.

1. Determine the next scheduled time as a unix epoch time from the "hours" specified by the user.
1b. To find 1, we need the value of today's midnight, as a unix epoch time.
2. Determine if a given time is within a window of the current time (now).

In addition, we can make this more clear by declaring constant values as perl constants. That helps because it shows intent to the programmer and the interpreter which things are not going to change as the program runs.

Here's some example code (not tested) that shows what I mean.

    use const SEC_PER_HOUR => 60 * 60;
    use const SEC_PER_DAY => 24 * 60 * 60;
    use const FLUCTUATION => 90;
    
    # Return the previous midnight time as a unix epoch time.
    sub
    midnight_time()
    {
       my $now = time(); 
       return $now - ($now % SEC_PER_DAY);
    }
    
    # Return the next scheduled time as a unix epoch time.
    sub
    schedule_time()
    {
       my $hour = shift;
       return midnight_time() + ($hour * SEC_PER_HOUR);
    }
    
    # Return true if target time is within the time window.
    sub
    in_window()
    {
        my $target = shift;
        my $now = time()
    
        my $range = (ALARM_INTERVAL / 2) + FLUCTUATION;
        my $lower = $now - $range;
        my $upper = $now + $range;
    
        return $lower < $target && $target <= $upper;
    }
    
    ....
    
        if (...) {
           if (in_window(schedule_time($CFG{EXPORT_HOUR}))) {
                                                                                                                                            2,5           Top